src/plugins/customBorders/customBorders.js

```diff
@@ -241,33 +230,20 @@ class CustomBorders extends BasePlugin {
     });
   }
 
-  /**
-   * Remove borders divs from DOM.
-   *
-   * @param {String} borderClassName Border class name as string.
-   */
-  removeBordersFromDom(borderClassName) {
-    let borders = this.hot.rootElement.querySelectorAll(`.${borderClassName}:not(td)`);
-
-    rangeEach(0, borders.length - 1, (index) => {
-      let parent = borders[index].parentNode;
-
-      if (parent.parentNode) {
-        parent.parentNode.removeChild(parent);
-      }
-    });
-  }
-
   /**
    * Remove border (triggered from context menu).
    *
    * @param {Number} row Visual row index.
    * @param {Number} col Visual column index.
    */
   removeAllBorders(row, col) {
-    let borderClassName = createClassName(row, col);
+    const borderId = createId(row, col);
```

Comment:
Could you use full names for arguments? Such as `col` to `column`?

Reply:
Done.